ssl_cert_tracker/new_tasks.py

```diff
@@ -7,9 +7,13 @@
 from celery import shared_task
 from libnmap.process import NmapProcess
 
-from ssl_cert_tracker.models import TestCertsData
+from ssl_cert_tracker.models import TestCertsData, TestCertsHistory
 from orion_integration.lib import OrionSslNode
 
+from .db_helper import Insert_Into_CertsData
+
+from .serializers import MyCertsDataSerializer
+
 logging.basicConfig(filename='pocProject.log',level=logging.DEBUG)
```

Comment:
why is this log file configuration hard coded here? both django and celery provide extensive logging facilities; please use them